crates/cubecl-linalg/src/matmul/components/global/args.rs

```diff
@@ -11,7 +11,9 @@ use crate::matmul::components::{self, MatmulPrecision, MatmulProblem, MatmulSele
 
 use super::Quantization;
 
-pub trait InputsLaunch: LaunchArg {
+/// Create the input runtime arguments for a matmul kernel that works on concrete inputs and
+/// output (not fused).
+pub trait ConcreteInputsFactory: LaunchArg {
```

Comment:
It is weird that a trait is named `ConcreteSomething` as trait are by nature abstract.

Maybe simply remove the prefix and use `InputsFactory` instead?